homeassistant/components/ituran/sensor.py

```diff
@@ -0,0 +1,111 @@
+"""Sensors for Ituran vehicles."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.const import (
+    DEGREE,
+    UnitOfElectricPotential,
+    UnitOfLength,
+    UnitOfSpeed,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import IturanConfigEntry
+from .coordinator import IturanDataUpdateCoordinator
+from .entity import IturanBaseEntity
+
+
+@dataclass(frozen=True)
+class IturanSensorEntityDescription(SensorEntityDescription):
+    """Describes Ituran sensor entity."""
+
+    enabled_by_default: bool = True
+
+
+SENSOR_TYPES: list[IturanSensorEntityDescription] = [
+    IturanSensorEntityDescription(
+        key="address",
+        translation_key="address",
+        enabled_by_default=False,
+    ),
```

Comment:
```suggestion
SENSOR_TYPES: list[SensorEntityDescription] = [
    SensorEntityDescription(
        key="address",
        translation_key="address",
        entity_registry_enabled_default=False,
    ),
```

Reply:
Or, rather extend the `IturanSensorEntityDescription` to have a `value_fn`. This way you can pass in a lambda to get data in a typesafe way. This avoids the use of `getattr`

Reply:
Added `value_fn` and matching type hints